streamly.cabal

```diff
@@ -125,6 +103,30 @@ extra-source-files:
     test/streamly-tests.cabal
     test/version-bounds.hs
 
+extra-doc-files:
+    CONTRIBUTING.md
+    Changelog.md
+    README.md
+    credits/*.md
+    credits/Yampa-0.10.6.2.txt
+    credits/base-4.12.0.0.txt
+    credits/bjoern-2008-2009.txt
+    credits/clock-0.7.2.txt
+    credits/foldl-1.4.5.txt
+    credits/fsnotify-0.3.0.1.txt
+    credits/hfsevents-0.1.6.txt
+    credits/pipes-concurrency-2.0.8.txt
+    credits/primitive-0.7.0.0.txt
+    credits/transient-0.5.5.txt
+    credits/vector-0.12.0.2.txt
+    dev/*.md
+    dev/*.png
+    dev/*.rst
+    docs/*.md
+    docs/*.hs
+    docs/*.svg
+    docs/API-changelog.txt
+
```

Comment:
Did you deliberately not add `benchmarks/*.md` here? Benchmarks is as extension of the current package and is ideally a doc for the devs and bench running enthusiasts?